README.md

```diff
@@ -10,41 +10,33 @@ Mock tests: ![Mock tests](https://flowcrypt.semaphoreci.com/badges/flowcrypt-bro
 
-This project is written in TypeScript. Browsers only understand JavaScript, so the project needs to be transpiled. You need to build the project the first time you download it, and build it after each change you make to see the result in the browser. First, install tooling:
+This project is written in TypeScript. Browsers only understand JavaScript, so the project needs to be transpiled. You need to build the project the first time you download it, and build it after each change you make to see the result in the browser. To get started, please follow the instructions below:
 
+1. Install the tooling by running the following command after navigating to the appropriate folder:
 ```bash
 $ cd some/folder/flowcrypt-browser
 $ npm install
 ```
+2. To build the project (currently available for Linux and Mac only), run the following command:
 
-To build the project (for now Linux and Mac only)
-
-```
-$ npm run-script build
-
-> flowcrypt-browser@5.9.7 build /home/luke/git/flowcrypt-browser
-> gulp
-
-[02:42:04] Using gulpfile ~/git/flowcrypt-browser/gulpfile.js
-[02:42:04] Starting 'default'...
-[02:42:04] Starting 'flush'...
-[02:42:04] Finished 'flush' after 62 ms
-[02:42:04] Starting 'transpileProjectTs'...
-[02:42:04] Starting 'copySourceFiles'...
-[02:42:04] Starting 'copyVersionedManifest'...
-[02:42:05] Finished 'copyVersionedManifest' after 199 ms
-[02:42:05] Finished 'copySourceFiles' after 557 ms
-[02:42:12] Finished 'transpileProjectTs' after 7.57 s
-[02:42:12] Starting 'chromeBuildSpacesToTabs'...
-[02:42:12] Finished 'chromeBuildSpacesToTabs' after 4.55 ms
-[02:42:12] Starting 'copyChromeToFirefox'...
-[02:42:12] Finished 'copyChromeToFirefox' after 323 ms
-[02:42:12] Starting 'copyChromeToFirefoxEditedManifest'...
-[02:42:12] Finished 'copyChromeToFirefoxEditedManifest' after 10 ms
-[02:42:12] Finished 'default' after 7.97 s
+```bash
+$ npm run build
 ```
+After running this command, you can find your built project in the build/chrome-consumer and build/firefox-consumer folders.
+
+To load the extension in Google Chrome, please follow these steps:
+
+1. Open your Chrome browser and navigate to chrome://extensions/.
+2. If not already enabled, toggle on the Developer mode switch located at the top-right corner.
+3. Click on the Load Unpacked button.
+4. Browse to and select the appropriate folder, either build/chrome-consumer or chrome-enterprise.
+
+Similarly, to load the extension in Firefox, follow these steps:
 
-Now you can find your built project in `build/chrome-consumer` and `build/firefox-consumer`
+1. Open your Firefox browser and navigate to about:debugging.
+2. Click on the "This Firefox" tab.
+3. Click on the "Load Temporary Add-on" button located at the top-right corner.
+4. Browse to and select the appropriate folder, either build/firefox-consumer or firefox-enterprise, and select the manifest.json file.
 
 Printing debug data to test logs can be done using special `Debug` class:
```

Comment:
In Firefox we can also run extension with `npm run run_firefox` command, should be added to `README` too
I usually use it, as in this case it runs in separate Firefox and doesn't mix with production extension installed.